charts/agent/Chart.yaml

```diff
@@ -2,30 +2,30 @@ apiVersion: v2
 name: agent
 description: Chart to install K8s collection stack based on Observe Agent
 type: application
-version: 0.8.0
+version: 0.8.1
 appVersion: "1.0.0"
 dependencies:
-  - name: opentelemetry-collector
```

Comment:
the space edits seems weird, maybe your linter is different?

Reply:
Yeah I found a weird setting in my formatter that makes lists not indented. Weird.